bank/client/rest/tx.go

```diff
@@ -45,9 +46,30 @@ func SendRequestHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
+		// amount cannot be zero
+		if req.Amount.IsZero() {
+			return
+		}
```

Comment:
We cannot return without writing on response otherwise API call will never complete.

Reply:
```
rest.WriteErrorResponse(w, http.StatusBadRequest, "invalid amount")
return
```